### PR TITLE
Problems solved on 10/9

### PR DESCRIPTION
**Pull request by HidemichiShimura**
These are problems on LeetCode which were solved on 10/9.

### CODE REVIEW

File: 242.valid-anagram.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        return false;
+    }
+    
+    for (let i = 0; i < sArr.length; i++) {
+        if (sArr[i] !== tArr[i]) {
+            return false;
+        }
+    }
+    
+    return true;
```
Comment (yuya-minamide) [inline 242.valid-anagram.js:25]:
```suggestion
    return sArr.join("") === tArr.join("");
```
You can also write like this.

**Review by yuya-minamide — APPROVED** [review]

File: 242.valid-anagram.js (hunk 1 above)
Comment (ryonryon) [reply 242.valid-anagram.js:25]: @yuya-minamide it looks good

**Review by ryonryon — APPROVED** [review]

File: 242.valid-anagram.js (hunk 1 above)
Comment (HidemichiShimura) [reply 242.valid-anagram.js]: I modified the code as the suggested change above.

**Comment by HidemichiShimura** [comment]
Thank you for your reviews

**Pull request merged**